src/tests/class_tests/openms/source/MzCalibration_test.cpp

```diff
@@ -171,82 +175,44 @@ START_SECTION(void compute(FeatureMap& features, const MSExperiment& exp))
 	feature1.setPeptideIdentifications(identifications);
 	fmap.push_back(feature1);
 	cal.compute(fmap_empty, exp);
-	std::cout << "\n Compute empty PeptideIdentification without error.\n";
+	TEST_EQUAL(fmap_empty.isMetaEmpty(), true);
 
 	//test empty hit
 	peptide_ID.setHits(std::vector<PeptideHit>{});
 	identifications.clear();
 	identifications.push_back(peptide_ID);
 	fmap.push_back(feature1);
 	cal.compute(fmap_empty, exp);
-	std::cout << "\n Compute empty PeptideHit without error.\n";
+	TEST_EQUAL(fmap_empty.isMetaEmpty(), true);
 
 	//test exception if RT-values of the feature map are greater than the greatest RT-values in MSexpriment
-	pre.setMetaValue("mz_raw", 4.4);
-	spec.setPrecursors({ pre });
-	spec.setRT(4);
-	spectra.push_back(spec);
-	exp.setSpectra(spectra);
-	
-	peptide_hit.setCharge(2);
-	peptide_hits.push_back(peptide_hit);
-	peptide_ID.setHits(peptide_hits);
-	peptide_ID.setRT(5);
-	identifications.push_back(peptide_ID);
-	feature1.setPeptideIdentifications(identifications);
-	fmap.clear();
-	fmap.push_back(feature1);
+	fmap[0].getPeptideIdentifications()[0].setRT(4);
 	TEST_EXCEPTION_WITH_MESSAGE(Exception::IllegalArgument, cal.compute(fmap, exp), "The retention time of the MZML and featureXML file does not match.");
 
 	//test exception if RT difference between mz_raw (get from MSexpriment) and mz after calibration (in FeatureMap) is greater than EPSILON_
-	pre.setMetaValue("mz_raw", 4.4);
-	spec.setPrecursors({ pre });
-	spec.setRT(4);
-	spectra.push_back(spec);
-	pre.setMetaValue("mz_raw", 2.4);
-	spec.setPrecursors({ pre });
-	spec.setRT(5);
-	spectra.push_back(spec);
-	exp.setSpectra(spectra);
-
-	peptide_ID.setRT(4.1);
-	identifications.clear();
-	identifications.push_back(peptide_ID);
-	feature1.setPeptideIdentifications(identifications);
-	fmap.clear();
-	fmap.push_back(feature1);
-	TEST_EXCEPTION_WITH_MESSAGE(Exception::IllegalArgument, cal.compute(fmap, exp), "PeptideID with RT " + to_string(4.1) + " s does not have a matching MS2 spectrum. Closest RT was " + to_string(5.0) + ", which seems too far off.\n");
+	fmap[0].getPeptideIdentifications()[0].setRT(0.1);
+	TEST_EXCEPTION_WITH_MESSAGE(Exception::IllegalArgument, cal.compute(fmap, exp), "PeptideID with RT " + to_string(0.1) + " s does not have a matching MS2 spectrum. Closest RT was " + to_string(0.5) + ", which seems too far off.\n");
```

Comment:
use `String(0.1)` etc.
Also 0.1 is not exaclty representable in binary, i.e. the resulting string might not be identical across platforms... a fraction of 2^x is safer, e.g. 0.25.